### ISSUE TITLE
"Format Document" throws Info: Mismatching table column sizes

### ISSUE THREAD

**Issue by bddenhartog** [darkriszty/MarkdownTablePrettify-VSCodeExt#10]
I have a `.md` file with a single table. Selecting that table and executing the `Format Selection` action (`Ctrl+K Ctrl+F` or through the context menu) formats that table correctly.

However, if I execute the `Format Document` action (either before or after formatting the table through the aforementioned steps), this plugin throws a message notification:

    [Info] Mismatching table column sizes.

**Comment by darkriszty** [comment]
Hi @bddenhartog,

I'm not sure if we're talking about a known behavior or a bug. At the moment the extension supports to format selected tables because it can't have any additional text in the selection. 
I've just tested the behavior to format documents when there is only a table in the entire file, and that is working with version 1.1 of the extension. 
The behavior to format all tables without having to select them will be implemented in issue #7. This means that it will automatically have support to format a document that has additional texts beside the table(s).

By single table you mean that there is 1 table and some other text in a markdown file, or just the table?
Can you please put an example content in this issue? 
Thanks!

**Comment by bddenhartog** [comment]
The _issue_ is simply that the notification is displayed. It seems that it only occurs when there is other content in the markdown file. Try using `Format Document` with the following two examples:

**Example _WITHOUT_ notification**

```
| heading 1    | another table heading | column3   |
|--------------|-----------------------|-----------|
| some content | more content          | even more |
```

**Example _WITH_ notification**

```
# my file header

some content

| heading 1    | another table heading | column3   |
|--------------|-----------------------|-----------|
| some content | more content          | even more |
```

**Comment by darkriszty** [comment]
Thank you for the example! - now I know that it is a known behavior. Currently the formatting is only supported if you select the entire table and use the `Format selection`. 

Support to format all tables in a file without selecting anything will be implemented via issue #7, so this is kind of a duplicate of that.

**Comment by darkriszty** [comment]
If it's okay with you, I would like to close this duplicate issue and use #7 to add more comments if needed.

**Comment by bddenhartog** [comment]
> Thank you for the example! - now I know that it is a known behavior. Currently the formatting is only supported if you select the entire table and use the Format selection.
> 
> Support to format all tables in a file without selecting anything will be implemented via issue #7, so this is kind of a duplicate of that.

Right, I understand that formatting the table is currently only supported by selecting the table and executing the `Format Selection` action, either through the context menu or `Ctrl+K Ctrl+F` (the default key bindings on non-OSX platforms).

That said, _I am not reporting the lack of table formatting through `Format Document` as an issue_. I am reporting the notification that is displayed as an issue. `Format Document` is shared by other extensions; if I use `Format Document`, _your_ extension shouldn't throw a notification in my face.

**Comment by darkriszty** [comment]
Thanks for the additional explanations, now I understand! :smiley:

**Comment by darkriszty** [comment]
Thank you @bddenhartog for reporting this. The issue is fixed and it's published as v1.1.1 to the marketplace.

It was not possible to differentiate between `Format Selection` on the entire file and `Format Document`, therefore the info message will not appear for `Format Selection` when the entire file content was included in the selection.
